CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
     - Fixes to `_convert_mp4` #834
     - Replace deprecated calls to `yaml.safe_load()` #834
 
+## [0.5.1] (February 27, 2024)
```

Comment:
This format implies a release. If so, items above (position fixes, database_settings) should be included under the 0.5.1 heading. I would hold off on release and lump these in with the above. I would also combine this PR with edits proposed in #852

Reply:
Sounds good, will do that